cylc/flow/parsec/validate.py

```diff
@@ -582,27 +584,42 @@ def _unquoted_list_parse(cls, keys, value):
         Examples:
             >>> list(ParsecValidator._unquoted_list_parse(None, '"1", 2'))
             ['1', '2']
```

Comment:
😤
```
>>> list(ParsecValidator._unquoted_list_parse(None, '1,"2",3'))
['1', '2', '3']

>>> list(ParsecValidator._unquoted_list_parse(None, '1, "2", 3'))
['1', '"2"', '3']
```

Reply:
We've discussed this somewhere recently, just trying to find evidence...

Reply:
Probably in relation to #5118